### PR TITLE
[13.0] [MIG] sale_order_invoicing_finished_task

### PR DESCRIPTION
**Pull request by Nikul-Chaudhary**

### CODE REVIEW

**Comment by carlosdauden** [comment]
Can you include https://github.com/OCA/sale-workflow/pull/1416 and fix tests?

File: sale_order_invoicing_finished_task/tests/test_sale_order_invoicing_finished_task.py [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+
+        # Click on toggle_invoiceable method
+        task.toggle_invoiceable()
+        self.assertEqual(self.sale_order.invoice_status, "no")
+
+        task.toggle_invoiceable()
+        self.assertEqual(self.sale_order.invoice_status, "to invoice")
+
+        # Make the invoice
+        self.sale_order.action_invoice_create()
```
Comment (carlosdauden) [inline sale_order_invoicing_finished_task/tests/test_sale_order_invoicing_finished_task.py:131]:
```suggestion
        self.sale_order._create_invoices()
```

**Comment by rousseldenis** [comment]
@Nikul-Chaudhary Could you check this for migration https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0#technical-method-to-migrate-a-module-from-120-to-130-branch ?

Pre-commit changes should appear in a separate commit. Thanks

**Comment by carlosdauden** [comment]
This PR was superseded

**Pull request closed without merging**